src/content/enterprise/install/upgrade.mdx

```diff
@@ -29,8 +17,97 @@ $ helm upgrade <your-release-name> okteto/okteto -f config.yaml --namespace=okte
 For example:
 ```console
 $ helm repo update
-$ helm upgrade enterprise okteto/okteto -f config.yaml --namespace=okteto --version 0.9.x
+$ helm upgrade okteto okteto/okteto -f config.yaml --namespace=okteto --version 0.9.x
 ```
 *You can use `helm ls` to find the name of your release.*
 
 Please review [the release notes](/docs/enterprise/install/releases/) before upgrading. New features, known issues, and configuration changes will be listed there.
+
+## Upgrading to 1.x
+
+The Okteto chart 1.0 comes with two breaking changes due to the following changes:
+
+- The chart has been renamed from `okteto-enterprise` to `okteto`. This means the Builkit Kubernetes service is recreated on upgrade and Buildkit won't be available until you update your DNS entries.
```

Comment:
```suggestion
- The helm chart has been renamed from `okteto-enterprise` to `okteto`. All the resources created by the chart will have `okteto` as their prefix instead of `okteto-enterprise`. This change will cause all resources to be recreated upon upgrade.  Depending on your cloud provider and your networking configuration, you might need to update the DNS entry of the [build service](/docs/enterprise/administration/configuration/#buildkit) after upgrading your Okteto instance.
```